assets/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@massds/mayflower-theme",
+  "version": "1.0.0",
+  "description": "Mayflower theming assets, including color variables, fonts, icons and other imagery, along with componentized scss used to style Mayflower markup",
+  "main": "index.js",
+  "scripts": {
+    "test": "stylelint",
+    "stylelint": "stylelint '**/*.scss' --syntax scss --formatter verbose",
+    "stylelint-fix": "stylelint '**/*.scss' --fix --syntax scss --formatter verbose"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/massgov/mayflower/tree/master/assets"
+  },
+  "keywords": [
+    "theme",
+    "scss"
+  ],
+  "author": "Massachusetts Digital Services (MDS)",
+  "license": "MIT",
+  "dependencies": {
+    "bourbon": "^5.1.0",
+    "bourbon-neat": "^1.7.2",
+    "normalize-scss": "^7.0.1"
```

Comment:
Same comment as above. Can you provide context for how/what `normalize-scss` is used for?

Reply:
I think that's just normalize stylings for different browsers. We can exclude this from the mayflower theme package